.github/workflows/integrate.yml

```diff
@@ -11,9 +11,11 @@ jobs:
     name: Build
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest]
+        os: [ ubuntu-latest, windows-latest ]
         node: [12, 14]
     runs-on: ${{ matrix.os }}
+    env:
+      NODE_OPTIONS: "--max-old-space-size=4096"
```

Comment:
I'd be _very_ interested in figuring out why we need 4 GB of memory for running our test suite as that seems... excessive 🙈 Did the OOM errors only occur on Windows?

Reply:
Oh, it actually seems to be the build failing 🤔

Reply:
4GB are probably too much. I've just used the ~rule of thumb~ Bayes' estimation with power law prior of doubling the current value 🤷 https://github.com/Siteimprove/alfa/runs/2728515736

We may want to tighten that amount, of course…